var/spack/repos/builtin/packages/root/package.py

```diff
@@ -127,6 +129,8 @@ class Root(CMakePackage):
     variant('mysql', default=False)
     variant('opengl', default=True,
             description='Enable OpenGL support')
+    variant('oracle', default=True,
```

Comment:
```suggestion
    variant('oracle', default=False,
```
I think this is a better default for most users of this recipe

Reply:
I agree. Although by default Oracle support is ON in ROOT: https://github.com/root-project/root/blob/master/cmake/modules/RootBuildOptions.cmake#L151